src/zenml/stack/flavor.py

```diff
@@ -34,6 +34,31 @@ def name(self) -> str:
             The flavor name.
         """
 
+    @property
+    def flavor_docs_url(self) -> str:
```

Comment:
Again, this should probably be optional for custom integrations? And maybe shouldn't by default be implemented like this which is wrong for custom flavors?

I guess returning `None` here and then when registering it in the flavor registry, we could do
`flavor.flavor_docs_url or compute_default_docs_url()`?

Also, do we need the `flavor_` prefix or is `docs_url` fine as well?

Reply:
docs_url is fine

Yeah you're right, we can generate this elsewhere, stil makes sense to have it as a filed though, so that users can link to their own docs.